### PR TITLE
Restructure markup order to correct tab focus order.

### PR DESCRIPTION
**Pull request by sg00dwin**
Fixes https://github.com/openshift/origin-web-console/issues/1580

tested across supported browsers
*note default Firefox requires `accessibility.tabfocus` added to its' `about:config`

### CODE REVIEW

File: app/views/projects.html [hunk 1]
```diff
@@ -120,6 +91,35 @@ <h2 class="h1">
...
+                                display-name="{{(project | displayName)}}"
+                                type-name-to-confirm="true"
+                                stay-on-current-page="true"
+                                alerts="alerts">
+                              </delete-link>
+                            </li>
+                          </ul>
+                        </div>
+                      </div>
                       </div>
```
Comment (spadgett) [inline app/views/projects.html]: Looks like we have two closing divs here or there is a problem with indentation

Comment (sg00dwin) [inline app/views/projects.html]: indention issue fixed

**Comment by rhamilto** [comment]
> Visually does it look the same?

LGTM

screen shot 2017-05-26 at 1 17 33 pm

**Comment by spadgett** [comment]
[merge][severity: bug]

**Pull request merged**